packages/browser/package.json

```diff
@@ -37,18 +37,19 @@
         "/lib"
     ],
     "devDependencies": {
+        "@reduxjs/toolkit": "^1.9.5",
```

Comment:
I would add `@reduxjs/toolkit` as a `peerDependency` and externalize it in Webpack. We should not include it with our package.

Reply:
I'm torn, I feel like it may not need to be a `peerDependency`. For the actual sdk, all we really need are the types from `redux` which is included in `@reduxjs/toolkit`. I have `@reduxjs/toolkit` included as a `devDependency` to provide the necessary types and also so I can create a redux store, reducer, and actions in the tests. Thoughts?

> externalize it in Webpack
Would you mind elaborating?

Reply:
If we use only types from redux, then I would ensure that they are imported just as types:
```
import type { Middleware, Action } from 'redux';
```

Can you do this on every import from redux?

Reply:
Fixed. I made sure they are only imported as types. I believe those are the only two type imports from redux.